src/media/css/site.styl

```diff
@@ -284,7 +284,11 @@ a:active {
     }
     section.full > div {
         padding: 30px 0;
-        width: $max-site-width;
+        max-width: $max-site-width;
+    }
+    .placeholder + .placeholder section.full,
```

Comment:
The simple fix for this is to just remove that padding in a media query at 1080px. It makes the state between 1050px and 1080px a little bit awkward, but not bad. The difficult fix is to refactor the layout of the page, pulling `.app-list-filter` to be siblings with `.app-list-page` instead of its child. That would potentially cause some regressions. Thoughts?